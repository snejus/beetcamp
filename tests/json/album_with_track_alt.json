{
  "@context": "https://schema.org",
  "@id": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 2798384948
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 4
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 3710011613
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 1614229409
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 2798384948
        },
        {
          "@type": "PropertyValue",
          "name": "digital_release_description",
          "value": "Digital Bonus.\r\nLive Wire (Roll Dann Remix 2)"
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a2798384948_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "FLD001 // Gareth Wild - Common Assault EP"
    },
    {
      "@id": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep#p2171282163",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 2171282163
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "p"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 1614229409
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Other Vinyl"
        },
        {
          "@type": "PropertyValue",
          "name": "image_ids",
          "value": [
            21656549,
            21712017,
            21712022,
            21712024
          ]
        },
        {
          "@type": "PropertyValue",
          "name": "is_music_merch",
          "value": true
        },
        {
          "@type": "PropertyValue",
          "name": "type_id",
          "value": 18
        }
      ],
      "description": "For the inaugural EP, FOLD resident Gareth Wild presents ‘Common Assault’. With metallic tubular synthesis, cut-throat percussion and rolling rhythms, this EP is an effective and essential piece of material demonstrating all angles of Techno, immersive to peak-time.\n\nThe release features three original tracks and two locked grooves from Gareth with stripped back, dark-side remix work from Madrid’s very own Roll Dann.",
      "image": [
        "https://f4.bcbits.com/img/0021656549_10.jpg",
        "https://f4.bcbits.com/img/0021712017_10.jpg",
        "https://f4.bcbits.com/img/0021712022_10.jpg",
        "https://f4.bcbits.com/img/0021712024_10.jpg"
      ],
      "musicReleaseFormat": "VinylFormat",
      "name": "FLD001 - Common Assault EP",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "includes_digital_download",
            "value": true
          }
        ],
        "availability": "SoldOut",
        "price": 12.0,
        "priceCurrency": "GBP",
        "priceSpecification": {
          "minPrice": 12.0
        },
        "url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep#p2171282163-buy"
      }
    },
    {
      "@id": "https://foldrecords.bandcamp.com/track/a1-gareth-wild-live-wire",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://foldrecords.bandcamp.com/track/a2-gareth-wild-live-wire-roll-dann-remix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://foldrecords.bandcamp.com/track/a3-gareth-wild-dds-locked-groove",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://foldrecords.bandcamp.com/track/aa1-gareth-wild-common-assault",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://foldrecords.bandcamp.com/track/aa2-gareth-wild-saturn-storm",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://foldrecords.bandcamp.com/track/aa3-gareth-wild-quadrant-locked-groove",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@type": "MusicGroup",
    "name": "Gareth Wild"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "03 Sep 2021 10:40:23 GMT",
  "datePublished": "02 Nov 2020 00:00:00 GMT",
  "image": "https://f4.bcbits.com/img/a2798384948_10.jpg",
  "keywords": [
    "London"
  ],
  "mainEntityOfPage": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
  "name": "FLD001 // Gareth Wild - Common Assault EP",
  "numTracks": 6,
  "publisher": {
    "@id": "https://foldrecords.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 1614229409
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 600
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 21655804
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 600
      }
    ],
    "description": "The institution of arts based in the heart of East London's, Canning Town. Home to music and creative studios and a 500 capacity state-of-the-art performance space.",
    "foundingLocation": {
      "@type": "Place",
      "name": "London, UK"
    },
    "image": "https://f4.bcbits.com/img/0021655804_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebSite",
        "name": "fold.london",
        "url": "http://www.fold.london"
      }
    ],
    "name": "FOLD RECORDS",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "a"
          }
        ],
        "name": "artists",
        "url": "https://foldrecords.bandcamp.com/artists"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://foldrecords.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "merch",
        "url": "https://foldrecords.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://foldrecords.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://foldrecords.bandcamp.com/track/a1-gareth-wild-live-wire",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2847020791
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M57S",
          "mainEntityOfPage": "https://foldrecords.bandcamp.com/track/a1-gareth-wild-live-wire",
          "name": "A1. Gareth Wild - Live Wire"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://foldrecords.bandcamp.com/track/a2-gareth-wild-live-wire-roll-dann-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1382985841
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M51S",
          "mainEntityOfPage": "https://foldrecords.bandcamp.com/track/a2-gareth-wild-live-wire-roll-dann-remix",
          "name": "A2. Gareth Wild - Live Wire ( Roll Dann Remix )"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://foldrecords.bandcamp.com/track/a3-gareth-wild-dds-locked-groove",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2410206971
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H00M20S",
          "mainEntityOfPage": "https://foldrecords.bandcamp.com/track/a3-gareth-wild-dds-locked-groove",
          "name": "A3. Gareth Wild - DDS [Locked Groove]"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://foldrecords.bandcamp.com/track/aa1-gareth-wild-common-assault",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2674898874
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M15S",
          "mainEntityOfPage": "https://foldrecords.bandcamp.com/track/aa1-gareth-wild-common-assault",
          "name": "AA1. Gareth Wild - Common Assault"
        },
        "position": 4
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://foldrecords.bandcamp.com/track/aa2-gareth-wild-saturn-storm",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 276309811
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M05S",
          "mainEntityOfPage": "https://foldrecords.bandcamp.com/track/aa2-gareth-wild-saturn-storm",
          "name": "AA2. Gareth Wild - Saturn Storm"
        },
        "position": 5
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://foldrecords.bandcamp.com/track/aa3-gareth-wild-quadrant-locked-groove",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1813148828
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H00M20S",
          "mainEntityOfPage": "https://foldrecords.bandcamp.com/track/aa3-gareth-wild-quadrant-locked-groove",
          "name": "AA3. Gareth Wild - Quadrant [Locked Groove]"
        },
        "position": 6
      }
    ],
    "numberOfItems": 6
  }
}
