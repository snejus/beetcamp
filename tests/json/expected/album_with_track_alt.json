[
  {
    "album": "Common Assault EP",
    "album_id": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "ep",
    "albumtypes": "ep",
    "artist": "Gareth Wild",
    "artist_credit": null,
    "artist_id": "https://foldrecords.bandcamp.com",
    "artist_sort": null,
    "asin": null,
    "catalognum": "FLD001",
    "comments": null,
    "country": "GB",
    "data_source": "bandcamp",
    "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
    "day": 2,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": null,
    "label": "FOLD RECORDS",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 11,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": null,
    "tracks": [
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 357,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Live Wire",
        "track_alt": "A1",
        "track_id": "https://foldrecords.bandcamp.com/track/a1-gareth-wild-live-wire",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 351,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Live Wire (Roll Dann Remix)",
        "track_alt": "A2",
        "track_id": "https://foldrecords.bandcamp.com/track/a2-gareth-wild-live-wire-roll-dann-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 20,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 6,
        "release_track_id": null,
        "title": "DDS [Locked Groove]",
        "track_alt": "A3",
        "track_id": "https://foldrecords.bandcamp.com/track/a3-gareth-wild-dds-locked-groove",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 315,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Common Assault",
        "track_alt": "AA1",
        "track_id": "https://foldrecords.bandcamp.com/track/aa1-gareth-wild-common-assault",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": null,
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 365,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Saturn Storm",
        "track_alt": "AA2",
        "track_id": "https://foldrecords.bandcamp.com/track/aa2-gareth-wild-saturn-storm",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": null,
        "genre": null,
        "index": 6,
        "initial_key": null,
        "length": 20,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 6,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Quadrant [Locked Groove]",
        "track_alt": "AA3",
        "track_id": "https://foldrecords.bandcamp.com/track/aa3-gareth-wild-quadrant-locked-groove",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2020
  },
  {
    "album": "Common Assault EP",
    "album_id": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep#p2171282163",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "ep",
    "albumtypes": "ep",
    "artist": "Gareth Wild",
    "artist_credit": null,
    "artist_id": "https://foldrecords.bandcamp.com",
    "artist_sort": null,
    "asin": null,
    "catalognum": "FLD001",
    "comments": "For the inaugural EP, FOLD resident Gareth Wild presents ‘Common Assault’. With metallic tubular synthesis, cut-throat percussion and rolling rhythms, this EP is an effective and essential piece of material demonstrating all angles of Techno, immersive to peak-time.\n\nThe release features three original tracks and two locked grooves from Gareth with stripped back, dark-side remix work from Madrid’s very own Roll Dann.",
    "country": "GB",
    "data_source": "bandcamp",
    "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
    "day": 2,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": null,
    "label": "FOLD RECORDS",
    "language": null,
    "media": "Vinyl",
    "mediums": 1,
    "month": 11,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": null,
    "tracks": [
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": "FLD001 - Common Assault EP",
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 357,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Live Wire",
        "track_alt": "A1",
        "track_id": "https://foldrecords.bandcamp.com/track/a1-gareth-wild-live-wire",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": "FLD001 - Common Assault EP",
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 351,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Live Wire (Roll Dann Remix)",
        "track_alt": "A2",
        "track_id": "https://foldrecords.bandcamp.com/track/a2-gareth-wild-live-wire-roll-dann-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": "FLD001 - Common Assault EP",
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 20,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 6,
        "release_track_id": null,
        "title": "DDS [Locked Groove]",
        "track_alt": "A3",
        "track_id": "https://foldrecords.bandcamp.com/track/a3-gareth-wild-dds-locked-groove",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": "FLD001 - Common Assault EP",
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 315,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Common Assault",
        "track_alt": "AA1",
        "track_id": "https://foldrecords.bandcamp.com/track/aa1-gareth-wild-common-assault",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": "FLD001 - Common Assault EP",
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 365,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Saturn Storm",
        "track_alt": "AA2",
        "track_id": "https://foldrecords.bandcamp.com/track/aa2-gareth-wild-saturn-storm",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Gareth Wild",
        "artist_credit": null,
        "artist_id": "https://foldrecords.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://foldrecords.bandcamp.com/album/fld001-gareth-wild-common-assault-ep",
        "disctitle": "FLD001 - Common Assault EP",
        "genre": null,
        "index": 6,
        "initial_key": null,
        "length": 20,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 6,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Quadrant [Locked Groove]",
        "track_alt": "AA3",
        "track_id": "https://foldrecords.bandcamp.com/track/aa3-gareth-wild-quadrant-locked-groove",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2020
  }
]
